cloudsim-plus/src/main/java/org/cloudbus/cloudsim/brokers/DatacenterBrokerAbstract.java

```diff
@@ -771,14 +774,37 @@ public void requestIdleVmDestruction(final Vm vm) {
         }
     }
 
+    public List<Cloudlet> requestVmDestruction(final Vm vm) {
```

Comment:
Since the method requests the VM destruction right away, without even sending a message,
it should be renamed to `destroyVm`. This way, we provide name consistency with the `Host.destroyVm`.
It should be moved to DatacenterBroker interface too.

Reply:
:+1:  done